### ISSUE TITLE
Expeditions Replacement

### ISSUE THREAD

**Issue by musicmanvr** [DeltaV-Station/Delta-v#936]
## The Problem
As it stands right now, it's fairly obvious that salvage as a while has become a sheer powergame with no intention of ever doing their job. The current changes fix the issue, but leave a sizeable gap where expeditions used to be that can still be filled by something else.

## ADVENTURE ZONE
New researchable science technology, **Downworld Teleporter** a gateway that can be opened by filling it with plasma. This gateway **REQUIRES** five crew members in order to activate, no more, no less. From the console, you can either select the standard randomly generated dungeon crawler, or **pre-generated** dungeons with a steep difficulty curve. Teamwork is not incentivized, it is REQUIRED if everyone wishes to come back alive.

Basically, it's goon's adventure zone but it requires you to have five people willing to work together.

**Comment by eeshaa1** [comment]
It should not require a certain amount of players nor have a player limit.

**Comment by musicmanvr** [comment]
> It should not require a certain amount of players nor have a player limit.

under no circumstance will I accept a solo run of adventure zone or duo run, it's reasonable to want to go higher but I am just directly never allowing people to waste the teleport with one person

**Comment by deltanedas** [comment]
> no intention of ever doing their job.

when im salv with exped i fulton back 8 *crates* of mats after first mining trip, maybe more following that

**Comment by Bonktrauma** [comment]
> > It should not require a certain amount of players nor have a player limit.
> 
> under no circumstance will I accept a solo run of adventure zone or duo run, it's reasonable to want to go higher but I am just directly never allowing people to waste the teleport with one person

What your not realizing is salvage on most maps is capped at 2-4

**Comment by DangerRevolution** [comment]
Is this issue not solved by mapping the gateways for Expeds?

**Comment by Bonktrauma** [comment]
> Is this issue not solved by mapping the gateways for Expeds?

Last I heard gateways were not being mapped

**Comment by UnicornOnLSD** [comment]
To make this clear, salvage's lack of a shuttle renders expeditions undoable as of now. Plus ALSO as it stands is the main way to mine reliably (and get uranium namely), unless you consider Velcro's recent PR that made asteroids SOMEWHAT worth salvage's time.
Expeditions do not need a replacement, and even so that wouldn't be a DV specific issue then.

**Comment by Axionyxx** [comment]
Expeditions are still possible, and with shipyards incoming, this isn't relevant.

Salvage doesn't need expeditions to survive. We're an MRP server, not mining station.
